### PR TITLE
[CP-998] I want to see a sync state

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-XXX]

**Description**

- Added sync info on menu
- added error sync modal on Overview

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

### CODE REVIEW

**Review by igorbezsmertnyi — DISMISSED** [review]

File: packages/app/src/connecting/components/connecting.component.tsx [hunk 1]
```diff
@@ -60,7 +60,13 @@ const Connecting: FunctionComponent<{
...
   useEffect(() => {
     const timeout = setTimeout(() => {
-      if (unlocked && loaded && syncInitialized) {
+      if (
+        unlocked &&
+        loaded &&
+        syncInitialized &&
+        (syncState === SynchronizationState.Cache ||
+          SynchronizationState.Loaded)
+      ) {
```
Comment (dkarski) [inline packages/app/src/connecting/components/connecting.component.tsx:69]: according to the reducer logic, the `syncInitialized` can be true only when `syncState` is set to `Cache` or `Loaded`. This logic no handle the Harmony 😉

File: packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx [hunk 2]
```diff
@@ -243,6 +249,13 @@ export const PureOverview: FunctionComponent<Props> = ({
     }
   }, [restoreDeviceState])
 
+  const history = useHistory()
+
+  const onRetry = () => {
+    // TODO: do some logic to retry connection
+    history.push(URL_ONBOARDING.connecting)
```
Comment (dkarski) [inline packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx:256]: 🟡 
is it TODO or not? 🤔 

One of your AC is _"The failure modal allows triggering sync again"_

**Review by dkarski — APPROVED** [review]

File: packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx (hunk 2 above)
Comment (wercix) [reply packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx]: Fixed in [6e9f68e](https://github.com/mudita/mudita-center/pull/642/commits/6e9f68e9693fe7bc9971fdb61d6aa68c65fe95c2)

File: packages/app/src/connecting/components/connecting.component.tsx (hunk 1 above)
Comment (wercix) [reply packages/app/src/connecting/components/connecting.component.tsx]: Fixed in [6e9f68e](https://github.com/mudita/mudita-center/pull/642/commits/6e9f68e9693fe7bc9971fdb61d6aa68c65fe95c2)

**Review by pawel-mudita — APPROVED** [review]

**Pull request merged**